viewing_party/party.py

```diff
@@ -1,23 +1,151 @@
 # ------------- WAVE 1 --------------------
 
+# I feel like there's a better way to test if title, genre, & rating are truthy 
+# but not sure how
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+    if title and genre and rating:
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    has_watched = [movie] 
+    user_data = {"watched" : has_watched }
+    return user_data
+
 
+def add_to_watchlist(user_data, movie):
+    wants_to_watch = [] 
+    user_data = {"watchlist" : wants_to_watch }
+    wants_to_watch.append(movie)

```

Comment:
Same comment as above, we don't need to reassign user_data to a dictionary with the key "watchlist".

We can simply write 
```python
user_data{"watchlist"].append(movie)
return user_data
```